hugo/content/docs/metrics.md

```diff
@@ -214,37 +50,129 @@ https://github.com/asynkron/protoactor-dotnet/issues/948
 | Name                                                    | Type      | Labels                                           |
 |---------------------------------------------------------|-----------|--------------------------------------------------|
 | protocluster_virtualactors                              | Gauge     | id, address, clusterkind                         |
-| protocluster_virtualactor_spawn_duration_seconds        | Histogram | id, address, clusterkind                         |
-| protocluster_virtualactor_requestasync_duration_seconds | Histogram | id, address, clusterkind, messagetype, pidsource |
+| protocluster_virtualactor_spawn_duration                | Histogram | id, address, clusterkind                         |
+| protocluster_virtualactor_requestasync_duration         | Histogram | id, address, clusterkind, messagetype, pidsource |
 | protocluster_virtualactor_requestasync_retry_count      | Count     | id, address, clusterkind, messagetype            |
-| protocluster_topology_events                            | Gauge     | id, address, membershiphashcode                  |
-| protocluster_resolve_pid_duration_seconds               | Histogram | id, address, clusterkind                         |
+| protocluster_members_count                              | Gauge     | id, address                                      |
+| protocluster_resolve_pid_duration                       | Histogram | id, address, clusterkind                         |
+
+## Getting started
+
+[Realtime map is using Proto.OpenTelemetry](https://github.com/asynkron/realtimemap-dotnet/blob/main/Backend/Program.cs#L20) and might be used as a working example.
```

Comment:
Could be bad idea to link to a linke number on main branch. Link to specific commit instead?